packages/flutter/lib/src/material/selection_area.dart

```diff
@@ -41,6 +41,7 @@ class SelectionArea extends StatefulWidget {
     this.focusNode,
     this.selectionControls,
     this.magnifierConfiguration,
+    this.onSelectionChange,
```

Comment:
Maybe this would be better as "onSelectionChanged" with a "d" in order to match the same callback in [EditableText](https://master-api.flutter.dev/flutter/widgets/EditableText/onSelectionChanged.html).